### PR TITLE
Org readme draft

### PR DESCRIPTION
**Pull request by afoncubierta**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,18 @@
-# .github
\ No newline at end of file
+# BiomedSci AI organization
```
Comment (vagenas) [inline README.md]: I would just use "BiomedSciAI" (no organization, no space)

File: README.md [hunk 2]
```diff
@@ -1 +1,18 @@
...
+## 🙋‍♀️ What is BiomedSciAI?
+
+Our mission is to accelerate scientific discovery in Biomedical Sciences through a variety of tools that can help scientists do their research quicker and more effectively.
+
+Whether you use population-scale data like EHRs, subcellullar-scale data like spatially resolved omics, or a combination of various modalities, we host tools that will help you make the most out of the data.
```
Comment (vagenas) [inline README.md]: I would use "most out of it" to avoid repetition of "data".

File: README.md [hunk 3]
```diff
@@ -1 +1,18 @@
...
+# BiomedSci AI organization
+## 🙋‍♀️ What is BiomedSciAI?
+
+Our mission is to accelerate scientific discovery in Biomedical Sciences through a variety of tools that can help scientists do their research quicker and more effectively.
+
+Whether you use population-scale data like EHRs, subcellullar-scale data like spatially resolved omics, or a combination of various modalities, we host tools that will help you make the most out of the data. 
+
+## 🌈 How can I get involved?
+
+If you want to contribute and get involved check out our guidelines and code of conduct.
```
Comment (vagenas) [inline README.md]: We don't have any yet. Do we plan to add these as part of this PR?

**Review by vagenas — DISMISSED** [review]
@afoncubierta, 

- The content should go to `profile/README.md`, as this is what will be displayed on the org landing page. See the [docs here](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile#adding-a-public-organization-profile-readme) (btw there is also the option for a `.github-private` repo, if we need some content only for the members)
- I would break the lines in the README source at 80 (or most e.g. 100) characters to maintain readability in various settings (they will still be rendered the same)
- You can find more comments in-line.

File: README.md (hunk 3 above)
Comment (afoncubierta) [reply README.md]: I was planning to merge only once we have the rest of the things in place

Comment (vagenas) [reply README.md]: FYI if you intend to add more things to the PR, it is best to [mark it as draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request#converting-a-pull-request-to-a-draft), to signalize it is not finalized.

**Review by akoseki2 — APPROVED** [review]

**Review by akoseki2 — APPROVED** [review]

**Pull request merged**